### ISSUE TITLE
Repo Cleanup

### ISSUE THREAD

**Issue by coolbreezechloe** [computeranonymous/computer#347]
Hello,

I notice there are a lot of branches that have no active work on them and have been fully merged into the master branch (well the default branch in your case seems to be gh-pages). I suggest deleting branches which are not needed. For those branches that have a single commit on them that has been sitting out there for a while we need to examine the changes on the branch and decide if it should be merged into the mainline or not. I will try to start this soon. Thirdly it is standard practice to name the mainline branch "master" and so I suggest we replace gh-pages branch (which appears to be the mainline) with master unless there is some reason not to. I am glad to help out with these activities and general maintenance of the repo and such.

So in summary:
1. remove inactive branches
2. merge or delete active branches that are stale
3. use master branch as mainline instead of gh-pages

**Comment by Skud** [comment]
We should not rename the gh-pages branch! This is s GitHub thing; if you have a branch with that name, they will use it as a static website for your repo, at yourname.github.io/reponame -- and this is where the Computer Anonymous website is actually hosted.

In other words, yes, there is a good reason for that being the default branch :)

**Comment by coolbreezechloe** [comment]
Ahhh, that explains what "gh" stands for. Sorry I'm not familiar with github as much in that regard I guess. I'm surprised they don't have an option to select the branch. Thanks for the tip. I remove my suggestion to rename the mainline branch, but other suggestions stand. So far I've seen that most of the branches that have stale commits on them contain changes which are already present on the mainline hence making the branch unnecessary.

**Comment by coolbreezechloe** [comment]
I am closing the issue since after talking with the maintainers there seems to be little interest in cleanup or even belief that it is a problem.
